src/main/daml/Daml/Finance/Util/Date/DayCount.daml

```diff
@@ -171,15 +185,14 @@ calcDcfAct365L fromDate toDate =
   in
     (/ intToDecimal nTotDaysY2) . intToDecimal $ subDate toDate fromDate
 
--- Calculate 30/360 day count fraction. This is also known as the '30/360 (ISDA)' or 'Bond Basis'
--- day count convention.
-calcDcf30360 : Date -> Date -> Decimal
-calcDcf30360 fromDate toDate =
+-- Calculate 30/360 or 30/365 day count fraction.
+calcDcf30Fix : Date -> Date -> Int -> Decimal
+calcDcf30Fix fromDate toDate daysPerYear =
```

Comment:
This is just a style thing, but I would write this function as

```suggestion
calcDcf30Fix daysPerYear fromDate toDate =
```

This way you can quickly define `calcDcf30360 = calcDcf30Fix 360`